index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css">
```

Comment:
the only thing I'm confused about with this PR is why you are linking out to the icon stylesheet directly if you are just importing the icons directly to your project files anyways?